packages/debug/src/browser/debug-session-options.ts

```diff
@@ -32,21 +32,22 @@ export class DebugCompoundRoot {
 }
 
 export interface TestRunReference {
-    controllerId: string,
-    runId: string
+    controllerId: string;
+    runId: string;
 }
 
 export interface DebugSessionOptionsBase {
-    workspaceFolderUri?: string,
-    testRun?: TestRunReference
+    workspaceFolderUri?: string;
+    testRun?: TestRunReference;
+    startedByUser?: boolean;
```

Comment:
The option `startedByUser` does not indicate the consequence of this option. For example without looking at the code it's not clear whether this means that the user already confirmed a "force" start.

Should we rename this to `checkForMultipleSessions` or `warnIfMultipleSessions`? Then it's clear what will happen if it's set to `true`.

Reply:
> For example without looking at the code it's not clear whether this means that the user already confirmed a "force" start.

Looking at the VS Code sources and seeing where they are set to true gave me quite a good understanding of how this should behave. 

> Should we rename

No, I prefer to keep it the way it is named in the VS Code. Every time there is a compatibility issue in Theia, we will look into VS Code. Naming things the same will make keeping things running and compatible easier.

Reply:
I did not check the VS code sources. If it's named the same there, then I'm fine with it :+1: